litellm/model_prices_and_context_window_backup.json

```diff
@@ -8015,7 +8015,7 @@
         "source": "https://www.databricks.com/product/pricing/foundation-model-serving",
         "metadata": {"notes": "Input/output cost per token is dbu cost * $0.070, based on databricks Llama 3.1 70B conversion. Number provided for reference, '*_dbu_cost_per_token' used in actual calculation."}
     },
-    "databricks/meta-llama-3.3-70b-instruct": {
+    "databricks/databricks-meta-llama-3-3-70b-instruct": {
```

Comment:
please don't delete models. This can potentially break a user's existing code. Instead add a new entry with this

Reply:
I can do that, but I don't understand how the existing code may work for any user? The key is incorrect so Databricks will return a 404 for this endpoint. I doubt any user depends on that?

Seems better to remove this model so that there won't be two databricks models for the same model name? (one which returns a 404 and one which works).

Reply:
See added context here: https://github.com/BerriAI/litellm/pull/8093#discussion_r1939779212